src/lib/router.js

```diff
@@ -2,6 +2,8 @@
 
 const Container = require('constitute').Container
 const makeRouter = require('koa-router')
+const Model = require('five-bells-shared').Model
+const CaseFactory = require('../models/db/case')
```

Comment:
I moved this logic from the controller, because I didn't want the controller to depend on the db code. Not sure this is the best place for it, but we can discuss.